QEfficient/src/_transformers/auto.py

```diff
@@ -42,7 +42,7 @@ def is_transformed(self) -> bool:
         return getattr(self.model, "qeff_transformed", False)
 
     @classmethod
-    def from_pretrained(cls, pretrained_model_name_or_path: str, *args, **kwargs):
+    def from_pretrained(cls, pretrained_model_path: str, *args, **kwargs):
```

Comment:
NO this is supposed to be using directly `HF` and allow users to directly use `QEFFAutoModelForCausalLM` , so download when only model_card is passed, don't change it's name, we want to keep it in sync with HF API.

Reply:
#### Addressing all the comments regarding `pretrained_model_name_or_path` here

I wasn't sure if this was by design. Which led to me making model_path a required argument and hence splitting it this into `pretrained_model_name` and `pretrained_model_path`

Can you elaborate on this?